lhotse/cut.py

```diff
@@ -181,20 +193,107 @@ def from_dict(data: dict) -> 'Cut':
         )
 
 
+@dataclass
+class PaddingCut:
+    id: str
+    duration: Seconds
+    num_frames: int
+    num_features: int
+
+    sampling_rate: int
+    use_log_energy: bool
+
+    @property
+    def supervisions(self):
+        return []
+
+    @property
+    def frame_shift(self):
+        return round(self.duration / self.num_frames, ndigits=3)
+
+    # noinspection PyUnusedLocal
+    def load_features(self, *args, **kwargs) -> np.ndarray:
+        value = np.log(EPS) if self.use_log_energy else EPS
```

Comment:
Just checking: are we normalizing the signal to [-1,1] before doing FFT, or doing the old HTK/Kaldi compatibility thing in [-32k,32k]?

Reply:
Yeah, we're in the [-1, 1] range. I'm open to extending the padding with more options. I was thinking specifically about accepting a specified energy value from the user and random noise padding. Lowest seen energy sounds good too.